help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md

```diff
@@ -112,8 +118,7 @@ If the details provided are valid, the UI displays a **Connected** status with a
 ### Fill in destination details {#destination-details}
 
 To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
-
-![Sample screenshot showing how to fill in details for Salesforce Marketing Cloud](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/destination-details.png)
+![Platform UI screenshot showing the destination details.](../../assets/catalog/email-marketing/salesforce-marketing-cloud-exact-target/destination-details.png)
 
 *  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
 *  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
```

Comment:
comment applies to row 125, **Customer name**

> This can be any value, however a value is mandatory.

As a reader, I'd like more information here. What exactly is this field? Are there any suggested values that I should fill in?

Reply:
Been updated by engineering this field has been removed.  Seems there was a bug within Platform and that's they had added this field. With the bug resolved they have removed customer name from customer data fields.